### PR TITLE
Improve contributing docs and remove authors file

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -62,25 +62,21 @@ Ready to contribute? Here's how to set up `conda-devenv` for local development.
...
     $ git clone git@github.com:your_name_here/conda-devenv.git
 
-3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+3. Create a new conda environment for developing::
 
-    $ mkvirtualenv conda-devenv
-    $ cd conda-devenv/
-    $ python setup.py develop
+    $ conda create -n devenv --file requirements_dev.txt
+    $ conda develop %CD%\conda_devenv
```
Comment (tadeu) [inline CONTRIBUTING.rst]: Cool 😄

Comment (gqmelo) [inline CONTRIBUTING.rst]: I would be nice not to give windows-only instructions

Comment (nicoddemus) [inline CONTRIBUTING.rst]: I thought about it... what do you suggest? Perhaps:

```rst
# Windows:
$ conda develop %CD%\conda_devenv

# Linux:
$ conda develop $PWD/conda_devenv
```

?

Comment (gqmelo) [inline CONTRIBUTING.rst]: As it is only this command I think it is good enough.

Comment (gqmelo) [inline CONTRIBUTING.rst]: Also, please add quotes around the path for both

Comment (tadeu) [inline CONTRIBUTING.rst]: Does `conda develop` need an absolute path? Couldn't it simply be `cd conda-devenv` and `conda develop .`?

Comment (nicoddemus) [inline CONTRIBUTING.rst]: Bingo!

**Pull request merged**